proposals/NNNN-SDL-behavior-in-case of-Low-Voltage_mqueue.md

```diff
@@ -0,0 +1,67 @@
+# SDL behavior in case of LOW_VOLTAGE event
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / Web / RPC]
+
+## Introduction
+
+This proposal defines SDL behavior when HMI requires SDL's services to be suspended due to a low voltage event.  
+A low voltage scenario is triggered when the battery voltage hits below a certain predefined threshold set by the system.  
+In case of such an event, the EMMC is turned off and all read/write operations are unavailable.  
+After the voltage level is restored all oprations are resumed.
+
+## Motivation
+
+Implement logic that will allow SDL to resume after battery charge is restored or to start up correctly in the next ingnition cycle if SDL was shut down due to LOW VOLTAGE event.  
```

Comment:
"if SDL was shut down due to LOW VOLTAGE event. " should be "if SDL was shut down due to a LOW VOLTAGE event. "

Reply:
@robinmk I propose to change to `if SDL functionality was suspended due to a LOW VOLTAGE event. ` as SDL process actually exists and not being shut down (just suspended and has limited functionality)
Do you agree?